### ISSUE TITLE
silk-cli doesn't work properly on Windows.

### ISSUE THREAD

**Issue by anefzaoui** [silklabs/silk#6]
User hunter-x on the community website reported that he can't get the Silk Command Line Interface to work properly on Windows, while this is hard for us to debug since we initially developed to run on development-based environments like Linux and OSX, it would make a good first bug to try and solve.

More on the issue: https://community.silklabs.com/t/unfortunately-i-cant-install-silk-cli-globally/52

**Comment by armisis** [comment]
Attached debug log from windows install attempt, it would appear that this fault may impact silk-emulator as well, shall I open a separate ticket for that? 

[npm-debug.txt](https://github.com/silklabs/silk/files/325308/npm-debug.txt)

Thanks,

Dave

**Comment by mvines** [comment]
Yes let's create a separate issue for Windows emulator support please. This'll be more involved than supporting the cli on Windows, as we currently have a bash dependency in our [run-emulator](https://github.com/silklabs/silk/blob/3132fc6f52ed2d5c700a4303b902176c253de4d4/bsp-gonk/board/qemu/archive.sh#L27) script and we [don't include](https://github.com/silklabs/silk/blob/3132fc6f52ed2d5c700a4303b902176c253de4d4/bsp-gonk/board/qemu/archive.sh#L12) the Windows qemu binary in the archive.  Rewriting run-emulator in node could be one way to easily support Windows here, if somebody is motivated enough to do it!

**Comment by nestornav** [comment]
Hi, 

I wrote in the forum about I couldn't install in my win 8.1 the silk-cli tool.  This is the [link](https://community.silk.io/t/unfortunately-i-cant-install-silk-cli-globally/52/7) in the forum.

Also I attached my error log [npm-debug.txt](https://github.com/silklabs/silk/files/619018/npm-debug.txt)

Thanks,
Nestor

**Comment by mvines** [comment]
Unfortunately there's still no native windows support for the silk-cli.  This may be a interim option for win 10 users though -- https://msdn.microsoft.com/en-us/commandline/wsl/about
